linux/zynqlinux/copy_to_sd_card.sh

```diff
@@ -6,6 +6,12 @@ if [[ -z "${SD_BOOT_PARTITION}" ]]; then
   SD_BOOT_PARTITION="/run/media/${USER}/ZYNQ_BOOT"
 fi
 
+if [[ ! -d "${SD_BOOT_PARTITION}" ]]; then
+  echo "Error: Could not open '${SD_BOOT_PARTITION}'!"
+  echo "Please make sure your SD card is mounted and adjust 'SD_BOOT_PARTITION' to its mount point."
+  exit 1
+fi
```

Comment:
else if [ $NOT_MOUNTED -eq 1 ]; then
  echo "Error: '$SD_BOOT_PARTITION' not mounted."
  exit 1
fi